blog.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
```

Comment:
Make sure and add the meta tag referenced in the first week to all your HTML pages.